examples/wind/src/layers/particle-layer/particle-layer.js

```diff
@@ -291,9 +290,11 @@ export default class ParticleLayer extends Layer {
 
     const modelTF = new Model(gl, {
       id: 'ParticleLayer-modelTF',
-      program: new ProgramTransformFeedback(gl, {
+      program: new Program(gl, {
         vs: vertexTF,
-        fs: fragmentTF
+        fs: fragmentTF,
+        varyings: ['gl_Position'],
+        bufferMode: gl.SEPARATE_ATTRIBS
```

Comment:
This is the default? Doesn't hurt to add it I suppose.

Reply:
Yeah, it is the default.